### DISCUSSION TITLE
Possibility of carrying out automated measurements only in sub-intervals of the scope acquisition

### DISCUSSION THREAD

**Discussion by PulseDoo** [fhdm-dev/scoppy#127] [Q&A]
Hello,
first of all congratulations for this application which I am using intensely.
From what I see it seems that the "automatic" measurements that can be carried out and shown on each channel (analogue and digital) always refer to the entire acquisition.
Would it be possible to configure the app\Scoppy to carry out and display automatic measurements (I mean, without the intervention of the cursors) in a sub-interval of the entire acquisition (e.g. with a definition based on the divisions of the time scale, such as from division 1 to division 3) ?

Thanks in advance

Kind regards

**Comment by fhdm-dev** [comment]
Hi,
Those measurements actually refer to the part of the waveform that visible on the screen and not the entire aquisition. 
Currently the app can't do what you want but I'll add your suggestion to my 'requested features' list (but I'm not sure if/when it will be implemented).

:smile:

**Comment by PulseDoo** [comment]
Hello,
many thanks for the quick feedback, the possibility of monitoring the measurements made on a portion of the acquired signal (e.g. one or two time divisions), as on many oscilloscopes, would be a valuable addition.
I'm a little confused though, doesn't what I see on the Oscilloscope display represent all the acquisition?
Thanks in advance

Kind regards

**Comment by fhdm-dev** [comment]
Only half the total acquisition is shown on the display.

**Comment by PulseDoo** [comment]
Hello,
thanks a lot for the feedback, so I understand that the application can somehow "buffer" an half part of the trace other than the one displayed, is this correct?
If so, how can I access it?
In any case this feature is not for me at all clear from the documentation (but maybe I missed some references)...
Kind regards.

**Comment by fhdm-dev** [comment]
Not really. It just captures more than what is displayed on the screen. If you hit STOP and then zoom out (or change the horizontal position) you can see the extra data. The portion of the capture buffer that is displayed on the screen is indicated at top of the display in the midde (the horizontal line with the rectangle). The horizontal line represents all of the data that has been captured. The rectangle is that part of the data that is currently displayed (tap one of the horizontal position buttons and you'll see it move).

**Comment by PulseDoo** [comment]
Hello,
many thanks for the feedback, it's clearer now.
I also found part of your explanations in the Scoppy application manual\documentation.
Kind regards.
